src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -65,3 +101,22 @@
 Error_Flattening_User_Supplied_Value_Dict = 'Error while flattening the user supplied helm values dict'
 Upgrade_RG_Cluster_Name_Conflict = 'The provided cluster name and rg correspond to different cluster'
 Corresponding_CC_Resource_Deleted_Fault = 'CC resource corresponding to this cluster has been deleted by the customer'
+Kubernetes_Node_Type_Fetch_Fault = 'Error while trying to find a linux/amd64 node for scheduling pods'
+Linux_Amd64_Node_Not_Exists = 'Kubernetes cluster doesnt have amd64/linux node'
+Kubernetes_Latest_Version_Fetch_Fault = 'Error while fetching the latest stable kubernetes release'
+Operate_RG_Cluster_Name_Conflict = 'The provided cluster name and rg correspond to different cluster being operated on'
+No_Features_Param_Provided = "No parmeters for feature update were specified with the command. Please run 'az connectedk8s {} --help' to check parameters available for {}"
+# Custom_Locations_Cluster_Connect_Enable_Conflict = 'Custom locations enabling with cluster connect disabled'
+Custom_Locations_Registration_Check_Fault_Type = "Error while checking resource provider registration of custom locations."
+Custom_Locations_OID_Fetch_Fault_Type = "Error while fetching oid for custom locations."
+Client_Details_Not_Provided_For_AAD_RBAC_Fault = 'Client ID or secret not provided for AAD RBAC'
+Successfully_Enabled_Features = 'Successsfully enabled features: {} for the Connected Cluster {}'
+Successfully_Disabled_Features = 'Successsfully disabled features: {} for the Connected Cluster {}'
+Error_enabling_Features = 'Error while updating agents for enabling features. Please run \"kubectl get pods -n azure-arc\" to check the pods in case of timeout error. Error: {}'
+Error_disabling_Features = 'Error while updating agents for disabling features. Please run \"kubectl get pods -n azure-arc\" to check the pods in case of timeout error. Error: {}'
+CLIENT_PROXY_VERSION = '1.1.0'
```

Comment:
>CLIENT_PROXY_VERSION [](start = 0, length = 20)

Does this have a correlation with the release version?

Reply:
It doesnt have any correlation. Release version is just the date at which we release. Whereas this is the version of the csp exe.